src/Export/Skills/act_int.txt

```diff
@@ -978,16 +978,16 @@ local skills, mod, flag, skill = ...
 		if breakdown then
 			local FRDamageTaken = {}
 			t_insert(FRDamageTaken, "Damage Taken per Cast")
-			t_insert(FRDamageTaken, s_format("^8=^7 %d^8 (Life) * ^7%d%%^8 (of Life taken as Chaos Damage)", life, activeSkill.skillData.SelfDamageTakenLife * 100))
```

Comment:
Was there a reason for this shifting of colour code and space? It is also not updated in the .lua files fyi.

Reply:
It has a wonky interaction with thousand separators, as the logic for that sees the color code as a number and can sometimes attempt to put a separator between the invisible code and the actual number, resulting in a separator before the number
![image](https://user-images.githubusercontent.com/36479307/208244858-b2979b9a-4f06-48ff-acee-f9279ca33484.png)

I will check out what I missed.

Reply:
Just re run the exporter if you have that setup otherwise.
![image](https://user-images.githubusercontent.com/31533893/208244980-a173c39a-4847-4d90-8831-a3ce1d8a46a0.png)

![image](https://user-images.githubusercontent.com/31533893/208244972-4a33de1e-156f-42d3-b9f5-1a790d51e0f8.png)

Reply:
I haven't set up exporter yet. There's also quite a few other instances of colour code+number format without a space in between, but I just replaced them in the files that I touched as to not expand the scope too much. There are also a few cases of code+number where a space character can't be inserted in between as it would change the format - that could only be fixed with changing thousand separator logic itself.